### PR TITLE
UpdateFromUpstream: imagefiles 2019-03-02 (b5a13fa4)

### PR DESCRIPTION
**Pull request by phcerdan**
Run the UpdateFromUpstream.sh script to extract upstream imagefiles
using the following shell commands.

$ git archive --prefix=upstream-imagefiles/ b5a13fa4 -- 
  imagefiles/build-and-install-curl.sh
  imagefiles/build-and-install-git.sh
  imagefiles/build-and-install-openssh.sh
  imagefiles/build-and-install-openssl.sh
  imagefiles/dockcross
  imagefiles/entrypoint.sh
  imagefiles/install-cmake-binary.sh
  imagefiles/install-gosu-binary.sh
  imagefiles/utils.sh
   | tar x

git://github/compare/85593886fd3ba09b1e31873f1f28c977fabf2871...b5a13fa4d2ea39fef64bb62de18ae46bf8c5d93c

### CODE REVIEW

**Comment by jcfr** [comment]
the centos 5 script need to be explicitly updated. See https://github.com/dockbuild/dockbuild/blob/master/centos5-devtoolset2-gcc4/build_scripts/build.sh

File: centos5-devtoolset2-gcc4/build_scripts/build.sh [hunk 1]
```diff
@@ -9,10 +9,8 @@ CPYTHON_VERSIONS="3.6.4"
 
 # openssl version to build, with expected sha256 hash of .tar.gz
 # archive
-OPENSSL_ROOT=openssl-1.0.2o
-# Hash from https://www.openssl.org/source/openssl-1.0.2o.tar.gz.sha256
-# Matches hash at https://github.com/Homebrew/homebrew-core/blob/1766321103d9780f6e38d3ac7681b8fa42cdca86/Formula/openssl.rb#L11
-OPENSSL_HASH=ec3f5c9714ba0fd45cb4e087301eb1336c317e0d20b575a125050470e8089e4d
+OPENSSL_ROOT=openssl-1.0.2r
+OPENSSL_HASH=ae51d08bba8a83958e894946f15303ff894d75c2b8bbd44a852b64e3fe11d0d6
```
Comment (jcfr) [inline centos5-devtoolset2-gcc4/build_scripts/build.sh]: Please, also update the comment after explicitly checking the hashes match.

**Comment by jcfr** [comment]
`Manually update openssl for centos5-devtoolset2-gcc4` -> `centos5-devtoolset2-gcc4: Update OpenSSL version`

File: centos5-devtoolset2-gcc4/build_scripts/build.sh (hunk 1 above)
Comment (phcerdan) [reply centos5-devtoolset2-gcc4/build_scripts/build.sh]: I agree. I removed the comment 30 seconds before your suggestion.

**Comment by phcerdan** [comment]
Renamed.

**Comment by phcerdan** [comment]
Added comment about checking the openssl hash from upstream.

Green!

**Pull request merged**